pstress/pstress-run-PXC57.conf

```diff
@@ -229,12 +229,12 @@ VALGRIND_CMD="valgrind --suppressions=${BASEDIR}/mysql-test/valgrind.supp --num-
 VALGRIND_ERRORS_FOUND=0
 
 ################################################################################
-# Crash Recovery must not be enabled with pquery3 runs                         #
+# Crash Recovery must not be enabled with pstress runs                         #
 ################################################################################
 CRASH_RECOVERY_TESTING=0
 
 ################################################################################
-# Must not be enabled with pquery3 runs                                        #
+# Must not be enabled with pstress runs                                        #
 ################################################################################
 QUERY_CORRECTNESS_TESTING=0
 QUERY_DURATION_TESTING=0
```

Comment:
Remove both QUERY_CORRECTNESS and QUERY_DURATION. Option does not exist anymore. Also remove the related comments